sdk/communication/Azure.Communication.Administration/tests/samples/Sample2_PhoneNumberAdministrationClient.cs

```diff
@@ -0,0 +1,471 @@
+            string phonePlanGroupId = phonePlanGroupsList.First(group => group.PhoneNumberType == PhoneNumberType.TollFree).PhonePlanGroupId;
+
+            #region Snippet:GetPhonePlansAsync
+            var phonePlans = client.GetPhonePlansAsync(countryCode, phonePlanGroupId, locale);
+            await foreach (var phonePlan in phonePlans)
+            {
+                Console.WriteLine($"Plan: {phonePlan.LocalizedName}, {phonePlan.LocationType}");
+            }
+            #endregion Snippet:GetPhonePlansAsync
+
+            var firstphonePlan = (await phonePlans.ToEnumerableAsync()).First();
+            var areaCode = firstphonePlan.AreaCodes.First();
+            var phonePlanId = firstphonePlan.PhonePlanId;
+
+            #region Snippet:ReservePhoneNumbersAsync
+            var reservationName = "My reservation";
+            var reservationDescription = "reservation description";
+            var reservationOptions = new CreateReservationOptions(reservationName, reservationDescription, new[] { phonePlanId }, areaCode);
+            reservationOptions.Quantity = 1;
+
+            var reservationOperation = await client.StartReservationAsync(reservationOptions);
+            await reservationOperation.WaitForCompletionAsync();
+            #endregion Snippet:ReservePhoneNumbersAsync
+
+            #region Snippet:PersistReservePhoneNumbersOperationAsync
+            var reservationId = reservationOperation.Id;
+
+            // persist reservationOperationId and then continue with new operation
+
+            var newPhoneNumberReservationOperation = new PhoneNumberReservationOperation(client, reservationId);
+            await newPhoneNumberReservationOperation.WaitForCompletionAsync();
+            #endregion Snippet:PersistReservePhoneNumbersOperationAsync
+
+            #region Snippet:StartPurchaseReservationAsync
+            var reservationPurchaseOperation = await client.StartPurchaseReservationAsync(reservationId);
+            await reservationPurchaseOperation.WaitForCompletionAsync();
+            #endregion Snippet:StartPurchaseReservationAsync
+
+            #region Snippet:ListAcquiredPhoneNumbersAsync
+            var acquiredPhoneNumbers = client.GetAllPhoneNumbersAsync(locale);
+
+            await foreach (var phoneNumber in acquiredPhoneNumbers)
+            {
+                Console.WriteLine($"Phone number: {phoneNumber.PhoneNumber}, activation state: {phoneNumber.ActivationState}");
+            }
+            #endregion Snippet:ListAcquiredPhoneNumbersAsync
+
+            var acquiredPhoneNumber = reservationOperation.Value.PhoneNumbers.First();
+
+            #region Snippet:ReleasePhoneNumbersAsync
+            //@@var acquiredPhoneNumber = "<acquired_phone_number>";
+            var phoneNumbers = new List<PhoneNumber> { new PhoneNumber(acquiredPhoneNumber) };
+            var phoneNumberReleaseOperation = client.StartReleasePhoneNumbers(phoneNumbers);
```

Comment:
you can inline:
```C#
client.StartReleasePhoneNumbers(new [] { new PhoneNumber(acquiredPhoneNumber) });
```

Reply:
@RezaJooyandeh I would prefer to leave second line simpler as this is for the MD sample and it is somewhat easier to those who is glancing over it to catch that release starts by passing some phoneNumbers, whatever type those are.
Thoughts?

Reply:
I have added a new method that takes a single phone number and used it on a sample